src/main/java/io/leonard/maven/plugins/jspc/JspcMojo.java

```diff
@@ -284,6 +284,12 @@ public class JspcMojo extends AbstractMojo {
    */
   @Parameter(defaultValue = "true")
   private boolean strictQuoteEscaping;
+  
+  /**
+   * Set this to 'true' to bypass compilation of JSP sources. 
+   */
+  @Parameter(defaultValue = "false", property = "jspc.skip")
+  private boolean skipJspc;
```

Comment:
```suggestion
  private boolean skip;
```

I think we can leave out the jspc part which you can derive form the context.

Reply:
yep ok will update

Reply:
done